src/plugins/blocksmith/components/image-block-modal/index.jsx

```diff
@@ -48,6 +49,7 @@ class ImageBlockModal extends React.Component {
       isLink: block.has('url'),
       isNsfw: block.get('is_nsfw'),
       isValid: true,
+      launchText: block.get('launch_text') || '',
```

Comment:
@jalopez13 - why set it to `empty` string here when you are setting it to `null` here? https://github.com/triniti/cms-js/pull/40/files#diff-6a425f9b08db8e535d7f40060e6dfcdcR91, change this line to:
```suggestion
      launchText: block.get('launch_text') || null,
```

Reply:
@arunkarnati 

Updated to null.